cmd/medius/common/registry.go

```diff
@@ -28,27 +28,31 @@ var staticRegistry = []Entry{
 		Artifacts: []api.Artifact{
 			centosstream.New("9", "x86_64", &docs.UserData{Username: "cloud-user"}, defaultEnvVariables("u1.medium", "centos.stream9")),
 			centosstream.New("9", "aarch64", &docs.UserData{Username: "cloud-user"}, defaultEnvVariables("u1.medium", "centos.stream9")),
+			centosstream.New("9", "s390x", &docs.UserData{Username: "cloud-user"}, defaultEnvVariables("u1.medium", "centos.stream9")),
```

Comment:
Hi @0xFelix , Sure, looking it.

Reply:
Hi @0xFelix , 
Yes, I have checked, and it's working. I am attaching the logs below for your reference. Thank you. 
[s390x-build.txt](https://github.com/user-attachments/files/17382483/s390x-build.txt)

Reply:
When I first executed `bin/medius images push --target-registry=localhost:5000 --dry-run=false --insecure-skip-tls --force --focus=ubuntu:22.04`, I encountered the following error:

`error introspecting image "quay.io/containerdisks/ubuntu:22.04": Error parsing manifest for image: choosing image instance: no image found in manifest list for architecture "s390x", variant "", OS "linux" `

Since I was trying to build it locally on s390x, I commented out the section of code in `cmd/medius/images/push.go` (from [line 291](https://github.com/kubevirt/containerdisks/blob/main/cmd/medius/images/push.go#L291) to [line 304](https://github.com/kubevirt/containerdisks/blob/main/cmd/medius/images/push.go#L304)) that checks the quay.io manifest and modified the function to return true. I have included the git diff output in the logs for your reference. After making this change, the execution was successful. I have also done a `docker pull `to verify the images (`ubuntu:22.04 and centos-stream:9`) created for the s390x architecture.

Reply:
Please let me know if you have any suggestions or specific requirements for this implementation. I appreciate your guidance! Thank you.

Reply:
> I commented out the section of code [...] that checks the quay.io manifest

There is a `--source-registry` flag to change the source registry. That might have helped, or for next time. 

Did you run `medius images verify` to verify the images? I'm interested to see if the built images were able to boot successfully.

Reply:
Hi @0xFelix ,
I am trying to execute `medius images verify` but facing some errors. I'm currently investigating the issue. I will keep you updated about the progress.

Thank you for your understanding!

Reply:
Hi @0xFelix, @lyarwood, 

I have successfully verified the locally built images for s390x after making the following adjustments and would like to kindly request your consideration in adding s390x support:

1. I used [this kubevirtci fork](https://github.com/chandramerla/kubevirtci) for starting the cluster with the `make cluster-up` command. This PR is raised by one of our team members to introduce s390x support in the kubevirtci repository.
2. I modified pipeline.sh, replacing the CirrOS focus image with ubuntu:22.04.

Please let me know if you have any questions. Thank you very much for your consideration!

Reply:
@aditi-sharma-1 apologies myself and @0xFelix have been out on PTO.

Are there any efforts to merge the changes of that fork into core `kubevirtci`?

Are there also any efforts to provide `s390x` hosts as CI runners that this project could use to verify these images?

Without this we can't merge these changes.

Reply:
Hi @lyarwood,
My teammate is actively working on the kubevirtci PR, and its progress can be tracked here: https://github.com/kubevirt/kubevirtci/pull/1316.

Regarding the availability of s390x machines, as mentioned by one of my teammates earlier (https://github.com/kubevirt/containerdisks/pull/182#issuecomment-2312558107), it will take some time to provide these resources to host CI runners. 

I have executed the pipeline.sh file with the locally built Ubuntu image, and it is working well. If you would like, I can share the logs for your review. 

Could you please let me know which additional test cases or files need to be executed for the CI process? I will also try to run these locally for the images to ensure comprehensive testing. 

Thank you!